src/main/scala/com/gu/githubapi/GitHubApi.scala

```diff
@@ -0,0 +1,59 @@
+package com.gu.githubapi
+
+import com.gu.config.Config.GitHubConfig
+import com.gu.githubapi.Conversion.{ RunningLiveAppDeployment, runningLiveAppDeployments }
+import com.gu.okhttp.SharedClient
+import io.circe.Decoder
+import io.circe.parser._
+import io.circe.generic.auto._
+import okhttp3.{ MediaType, Request, RequestBody }
+
+import scala.util.Try
+
+object GitHubApi {
+
+  val graphQlApiUrl = "https://api.github.com/graphql"
```

Comment:
It might seem like overkill to use two different GitHub APIs, but this new API allows us to get deployment information with a single HTTP call: https://developer.github.com/v4/#why-github-is-using-graphql.

Unfortunately the [write operation](https://developer.github.com/v4/mutation/createdeploymentstatus/) we'll need is still under development in v4, so we'll need to use the old [REST API](https://developer.github.com/v3/repos/deployments/#create-a-deployment-status) for that.